### DISCUSSION TITLE
Any updates on Recipinned

### DISCUSSION THREAD

**Discussion by PeterJT** [mealie-recipes/mealie#3308] [General]
I was running mealie locally until a house fire destroyed everything :(

Any updates as to how Recipinned is progressing?

**Comment by hay-kot** [comment]
Hey! 

I can say that I've made lots of progress behind the scene. I do have a preview instance opened and have been collecting some feedback from users. Right now I have 5-6 more things on my to do list before I start a more formal Alpha release. I'm shooting for about a month from now for a soft release to a couple key people. Then a wider release sometime in the summer.

If you're interested in participating in the Alpha or helping with some feedback, please reach out on discord and let me know! I'm `haykot` on discord

Some key features I _have_ built currently

- Full Meal Planner w/ ical integration
- Advanced Search with better scoring and filtering than Mealie
- Cross "Group" sharing mechanism that is super flexible
- Public/Private sharing links to share recipes, or email someone.
- Recipe/Ingredient parsing is greatly improved and mostly automatic w/ user verification. 
- Improved cooking mode features for more automatic implementation
